### ISSUE TITLE
[Feature request] Allow the display of purchased ONS names instead of partial hash next to user name

### ISSUE THREAD

**Issue by nikmit** [oxen-io/session-desktop#2511]
Since user names are not unique but ONS names are, allowing the display of purchased ONS names next to the username would make it harder for a user to be impersonated in a public group. So instead of seeing
**maindev** (12fe...83ab)
 people would see
**maindev** (maindev) or **maindev busy** (maindev)
if the user bought the respective ONS and configured it in settings. The chosen handle may or may not be required to match the ONS name, I would suggest it should not be required to match.

As a side effect, this should create additional demand for ONS names.

**Comment by KeeJef** [comment]
Yeah we definitely want to change the way we display ONS, vs Nickname, vs Display names in the future, with ONS being clearly distinguished
